mantle/cmd/kola/options.go

```diff
@@ -248,6 +250,10 @@ func syncOptionsImpl(useCosa bool) error {
 		if err := syncCosaOptions(); err != nil {
 			return err
 		}
+	} else if kola.Options.Stream != "" {
```

Comment:
Hmm, seems like this would belong better as part of the cascading if-statements above instead? E.g.

```go
if kola.Options.CosaBuildId != "" {
  ...
} else if kola.Options.Stream != "" {
  ...
} else {
  // try auto-detection of cosa here
}
```

That way, `--stream` takes precedence over auto-detected cosa.

Reply:
Good suggestion, done!

Reply:
Forgot a `git push`? :)

Reply:
Indeed I did, fixed!